capstone/src/main/webapp/document_analyzer.html

```diff
@@ -0,0 +1,90 @@
+            <a class="nav-link" href="about.html">About</a>
+          </nav>
+        </div>
+      </header>
+      <h2>Attach a document and select the Perspective API attribute(s) you would like to be shown</h2>
+      <div class="input-group">
+        <div class="custom-file">
+          <input type="file" class="custom-file-input" id="documentForAnalysis" aria-describedby="inputFileText">
+          <label class="custom-file-label" for="documentForAnalysis">Select Document to be Analyzed</label>
+        </div>
+      </div>
+      <form>
+        <div class="form-check">
+          <input type="checkbox" class="form-check-input" id="toxicity">
+          <label class="form-check-label" for="toxicity">Toxicity</label>
+        </div>
+        <div class="form-check">
+          <input type="checkbox" class="form-check-input" id="identity-attack">
+          <label class="form-check-label" for="identity-attack">Identity Attack</label>
+        </div>
+        <div class="form-check">
+           <input type="checkbox" class="form-check-input" id="insult">
+          <label class="form-check-label" for="insult">Insult</label>
+        </div>
+        <div class="form-check">
+          <input type="checkbox" class="form-check-input" id="profanity">
+          <label class="form-check-label" for="profanity">Profanity</label>
+        </div>
+        <div class="form-check">
+          <input type="checkbox" class="form-check-input" id="threat">
+          <label class="form-check-label" for="threat">Threat</label>
+        </div>
+        <div class="form-check">
+          <input type="checkbox" class="form-check-input" id="sexually-explicit">
+          <label class="form-check-label" for="sexually-explicit">Sexually Explicit</label>
+        </div>
+        <div class="form-check">
+          <input type="checkbox" class="form-check-input" id="flirtation">
+          <label class="form-check-label" for="flirtation">Flirtation</label>
+        </div>
+        <button type="submit" class="btn btn-primary">Submit</button>
+      </form>
+      <div id="perspective-response"></div>
+      <footer class="mastfoot mt-auto">
+        <div class="inner">
+          <p>Built by Christopher Mohri, Daniel Ortega and Oluwadamilare Alamutu.</p>
+        </div>
+      </footer>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" 
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous">
+    </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" 
+      integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous">
+    </script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" 
+      integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous">
+    </script>
+  </body>
+</html>
```

Comment:
nit: Let's add a newline at the end of this file so the red symbol goes away.

Reply:
addressed in "add newline"